File: Python/beginner/1132.py
# https://judge.beecrowd.com/pt/problems/view/1132
# Múltiplos de 13

x = int(input())
y = int(input())

if x > y:
    print(sum([i for i in range(y, x + 1) if i % 13 != 0]))
else:
    print(sum([i for i in range(x, y + 1) if i % 13 != 0]))
    
# By Fernando Serra
# https://github.com/fernandosserra

File: Haskell/beginner/1132.hs
-- https://judge.beecrowd.com/pt/problems/view/1132
-- Múltiplos de 13

-- Importing ReplicateM
import Control.Monad (replicateM_)

-- Main
main = do
    -- Getting Interval
    x <- readLn
    y <- readLn
    -- Testing Interval
    if x > y
        then
            -- Testing conditions, filling acum and put it into screen
            print (sum [i | i <- [y..x], i `mod` 13 /= 0])
        else
            print (sum [i | i <- [x..y], i `mod` 13 /= 0])

-- By Fernando Serra
-- https://github.com/fernandosserra